core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -73,6 +90,16 @@ oppia.directive('collectionNodeCreator', [function() {
           );
         };
 
+        var cleanExplorationId = function(newExplorationId) {
```

Comment:
@seanlip -- Sure. Currently when we search for an exploration, results are displayed in this form: `(#4) Three Balls`.
Also, we add an exploration to a collection using it's id. So, somehow we need the id of the exploration so that we can pass it to `addExplorationToCollection`. This function accepts that string format and extracts the id of exploration.